staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -232,6 +243,19 @@ enum NamespaceMode {
     TARGET    = 3;
 }
 
+// UserNamespace describes the intended user namespace configuration for a sandbox.
+message UserNamespace {
+    // User namespace for this sandbox.
+    // Note: It currently supports only POD and NODE.
+    NamespaceMode mode = 1;
```

Comment:
Is it planned for the future to support the other modes as well or would this not make any sense at all? If not, then I'd suggest to use an inline enum to exclusively allow `Pod` and `Node` rather than reusing the enum `NamespaceMode`.